packages/styles/footer.import.styl

```diff
@@ -0,0 +1,41 @@
+footer
+  position absolute
+  bottom 0
+  width 100%
+  height $footer-height
+  z-index 100
```

Comment:
I've added three layers above the document base: 
10, 100, 1000 `$bottom-layer $mid-layer $top-layer`